code/index.html

```diff
@@ -1,30 +1,127 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <meta charset="utf-8" />
+    <meta charset="utf-8" /> <meta name="viewport" content="width=device-width, initial-scale=1">
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
+ <link href="/your-path-to-fontawesome/css/all.css" rel="stylesheet">
+ <link rel="stylesheet" href="style.css">
+ <link rel="stylesheet" href="script.js">
   </head>
   <body>
-    <!-- video or image as a header is cool :) -->
 
-    <!-- Signup form -->
+      <header>
+        <div class="topnav">
 
-    <section class="accordion">
-      <!-- Accordion -->
-      <h1>FAQ</h1>
+          <!-- Centered link -->
+          <div class="topnav-centered">
+            <img src="images/clogo.jpeg" alt="logo">
+          </div>
+        
+          <!-- Left-aligned links (default) -->
+          <a href="#news">Necklace</a>
+          <a href="#contact">Bracelete</a>
+          <a href="#contact">Earings</a>
+          <a href="#contact">Rings</a>
+        
+          <!-- Right-aligned links -->
+          <div class="topnav-right">
+            <a href="#search">Search</a>
+            <a href="#about">About us</a>
+          </div>
+</header>
 
-      <div class="question" id="section1">Section 1</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
-      </div>
+    <img src="images/armband.webp" alt="Bracelete">
+   
+<div class="form">
+      <div class="tab-content">
+        <h1 id="sign-up">Sign up for newsletter</h1>
+          <form action="/" method="post">
+            <div class="top-row">
+              <div class="field-wrap">
+                <label>
+                  <span class="req"></span>
+                </label>
+                <input type="text" required autocomplete="off" placeholder="First Name" />
+              </div>
+          <div class="field-wrap">
+                <label>
+                <span class="req"></span>
+                </label>
+                <input type="text"required autocomplete="off" placeholder="Last name"/>
+              </div>
+            </div>
+             <div class="field-wrap">
+              <label>
+              <span class="req"></span>
+              </label>
+            <input type="email" name="email" placeholder="Email Adress"/>
+            </div>
+            <div class="field-wrap">
+              <label>
+                <span class="req"></span>
+              </label>
+              <input type="password"required autocomplete="off" placeholder="Set A Password"/>
+            </div>
+            <button type="submit" class="button button-block"/>Get Started</button>  
+</form>
+</div>   
+</form>
+</div>
+</div>
+</div>
+</div>       
```

Comment:
It would be great if these closing tags is indented, then you will easier see where it opens! 😀